### DISCUSSION TITLE
Feature Request: Execute multiple statements at once in Node

### DISCUSSION THREAD

**Discussion by dsmilkov** [duckdb/duckdb#5629] [Ideas]
Same as https://github.com/duckdb/duckdb/issues/1100 and https://github.com/duckdb/duckdb/issues/400 but for the Node.js client.

Node example :
```typescript
connection.all(`
  CREATE VIEW integers AS SELECT * FROM range(10);
  SELECT * from integers;
`, (err, tableData) => {...})
```

Likewise for WASM:
```typescript
await connection.query(`...`);
```

Current: `Invalid Error: Cannot prepare multiple statements at once!`
Expected: It returns the result from the last statement.

**Comment by dsmilkov** [comment]
A question for the owners: should this live as an Issue since it's about API parity across languages instead of a Discussion?

**Reply by hannes** [reply]
No, this is a great place for this.

**Reply by hannes** [reply]
Can you make a proposal on how the API should work perhaps?

**Reply by dsmilkov** [reply]
Yup, I just edited my original post.
